osu.Desktop/OsuGameDesktop.cs

```diff
@@ -138,6 +138,7 @@ public override void SetHost(GameHost host)
                 // SDL2 DesktopWindow
                 case DesktopWindow desktopWindow:
                     desktopWindow.CursorState.Value |= CursorState.Hidden;
+                    desktopWindow.SetIconFromStream(Assembly.GetExecutingAssembly().GetManifestResourceStream(GetType(), "lazer.ico"));
```

Comment:
Might as well hoist out the resource stream to a local, if it's being used in both branches anyway.